test/petab/test_petab_import.py

```diff
@@ -76,6 +77,32 @@ def test_3_optimize(self):
                 result.optimize_result.get_for_key('fval')[0]))
 
 
+def test_plist_mapping():
+    """Test that the AMICI objective created via PEtab correctly maps
+    gradients entries when some parameters are not estimated (realized via
+    edata.plist)."""
+    model_name = "Boehm_JProteomeRes2014"
```

Comment:
boehm has 2 fixed parameters at -1, -5, therefore this should be a good example :+1: